app/routes/upload-image.tsx

```diff
@@ -94,10 +93,14 @@ export default function SetupPage() {
         encType="multipart/form-data"
         onSubmit={handleUpload}
       >
-        <input type="file" name="file" onChange={handleChange} />
+        <input
+          type="file"
+          name="file"
+          accept="image/*"
+          onChange={handleChange}
+        />
         <button
           type="submit"
-          accept="image/*"
```

Comment:
I saw this one and fixed it but hadn't committed it. Can tell my brain was slowing down. (Also apologies for not implementing any TypeScript in my commit).